src/plugins/dragresize_ie11.js

```diff
@@ -989,10 +989,15 @@
 			const imageStyles = image.getAttribute('style');
 
 			if (imageStyles) {
+				const heightStyles = /(height:.+?;)/g.exec(imageStyles);
+				const heightStyle = heightStyles[0];
+
 				const widthStyles = /(width:.+?;)/g.exec(imageStyles);
 				const widthStyle = widthStyles[0];
```

Comment:
We looked at this in a bit more detail and saw that it was possible to trigger an error here because we don't check whether `widthStyles` is `null` before accessing `[0]` on it (ie. a bug that pre-existed before this PR, but we add another instance of the same problem here). To see that happen, just set a width but not a height style (or vice a versa) and see the error in the console. So @julien is preparing a follow-up that guards against that case.